src/NovelAudioService.cpp

```diff
@@ -7,28 +7,26 @@
 
 namespace NovelRT {
 
-NovelAudioService::NovelAudioService() : _nextChannel(1), _musicTime(0), _musicPausedTime(0), isInitialized(false) {
+NovelAudioService::NovelAudioService() : _nextChannel(1), _musicTime(0), _musicPausedTime(0), isInitialized(false), _console(NovelUtilities::CONSOLE_LOG_AUDIO) {
   initializeAudio();
 }
 
 bool NovelAudioService::initializeAudio() {
   if (SDL_InitSubSystem(SDL_INIT_AUDIO) < NovelUtilities::SDL_SUCCESS)
   {
-    std::cout << "ERROR: Cannot play audio!" << std::endl;
-    std::cout << "SDL Error: " << SDL_GetError() << std::endl;
+    _console.log("Cannot play audio: " + std::string(SDL_GetError()) , LogLevel::ERR);
```

Comment:
nit: It might be nice for us to declare a helper function like: `LogIfSdlFailure(SDL_InitSubSystem(SDL_INIT_AUDIO), "Cannot play audio")`

That would make the methods smaller and would isolate/make consistent the error retrieval and formatting

Reply:
It would also allow more customization on what happens for particular failure kinds (as some of these should likely be throwing rather than just logging and continuing).

Reply:
May need to discuss this in Discord perhaps. Not sure how to go about handling this (properly).